README.md

```diff
@@ -28,7 +28,7 @@ Please don't use Github issues for feature requests, instead create a topic in t
 
 ### License
 
-Copyright 2014 Hummingbird Media, Inc.
+Copyright 2015 Hummingbird Media, Inc.
```

Comment:
I did read a couple of these before asking, actually. I was pondering over the question in regards my own projects just yesterday. They all mention that previous years, especially the first year of publication is significant. Hence I came to think that one should keep it.